bundle/cmake/GTClangCloneRepository.cmake

```diff
@@ -0,0 +1,93 @@
+##===------------------------------------------------------------------------------*- CMake -*-===##
+##                         _       _
+##                        | |     | |
+##                    __ _| |_ ___| | __ _ _ __   __ _ 
+##                   / _` | __/ __| |/ _` | '_ \ / _` |
+##                  | (_| | || (__| | (_| | | | | (_| |
+##                   \__, |\__\___|_|\__,_|_| |_|\__, | - GridTools Clang DSL
+##                    __/ |                       __/ |
+##                   |___/                       |___/
+##
+##
+##  This file is distributed under the MIT License (MIT). 
+##  See LICENSE.txt for details.
+##
+##===------------------------------------------------------------------------------------------===##
+
+include(CMakeParseArguments)
+
+# gtclang_clone_repository

```

Comment:
Again, move to MSBuild

Reply:
same argument about having to clone before getting access to msbuild. Let me know if this is fine